### PR TITLE
Small improvements in README and ServerInitialisation

### PR DESCRIPTION
**Pull request by kaspergrubbe**
Continuing from here: https://github.com/aquasync/ruby-vnc/pull/5, the only difference is the branch name (no longer from `kaspergrubbe/ruby-vnc:master`.

### CODE REVIEW

**Comment by aquasync** [comment]
Hi! Sorry I never got back to you on #5. I guess the only reason I didn't pull that one is I'd prefer to squash the pair of commits that gets reverted. That ok with you?

**Comment by kaspergrubbe** [comment]
@aquasync, oh that wasn't clear, no worries about the delay :) I have squashed the commits, and dealt with the merge conflict. Are we clear to merge?

File: lib/net/vnc.rb [hunk 1]
```diff
@@ -133,9 +133,12 @@ def connect
 			socket.write((options[:shared] ? 1 : 0).chr)
 
 			# ServerInitialisation
+			framebuffer_width  = socket.read(2).to_s.unpack('n')[0].to_i
```
Comment (aquasync) [inline lib/net/vnc.rb:136]: Not entirely clear on what the to_s and to_i are for - is the idea that if the socket is eof then framebuffer_width will be 0 (nil => '' unpacked to nil => 0?). Could perhaps just do `width, height = socket.read(4).unpack('n2')`, but maybe when your helper things land this gets replaced with those anyway...

Comment (kaspergrubbe) [reply lib/net/vnc.rb:136]: Ouch, that is a good point. I will see what I can do.

**Pull request merged**